.github/workflows/tests.yml

```
@@ -0,0 +1,22 @@
+name: Run tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: ['1.0', '1.1', '1.2', '1.3', '1.4', '1', 'nightly']
+        julia-arch: [x64, x86]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        exclude:
+          - os: macOS-latest
+            julia-arch: x86
```

```suggestion
            julia-arch: x86
     fail-fast: false
```
Otherwise a single failing job will cancel all other jobs.